ERROR - 2022-03-26 23:37:55 --> Error connecting to the database: No connection could be made because the target machine actively refused it
CRITICAL - 2022-03-26 23:37:55 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
#0 C:\xampp\htdocs\ci4App\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\ci4App\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\ci4App\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\ci4App\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\ci4App\app\Validation\Userrules.php(13): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\ci4App\system\Validation\Validation.php(292): App\Validation\Userrules->validateUser('12345678', 'email,password', Array, NULL)
#6 C:\xampp\htdocs\ci4App\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array)
#7 C:\xampp\htdocs\ci4App\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ci4App\app\Controllers\UserController.php(32): CodeIgniter\Controller->validate(Array, Array)
#9 C:\xampp\htdocs\ci4App\system\CodeIgniter.php(832): App\Controllers\UserController->login()
#10 C:\xampp\htdocs\ci4App\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#11 C:\xampp\htdocs\ci4App\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4App\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ci4App\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2022-03-26 23:38:15 --> Error connecting to the database: No connection could be made because the target machine actively refused it
CRITICAL - 2022-03-26 23:38:15 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
#0 C:\xampp\htdocs\ci4App\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\ci4App\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\ci4App\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\ci4App\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\ci4App\app\Validation\Userrules.php(13): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\ci4App\system\Validation\Validation.php(292): App\Validation\Userrules->validateUser('12345678', 'email,password', Array, NULL)
#6 C:\xampp\htdocs\ci4App\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('password', 'password', '12345678', Array, Array)
#7 C:\xampp\htdocs\ci4App\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ci4App\app\Controllers\UserController.php(32): CodeIgniter\Controller->validate(Array, Array)
#9 C:\xampp\htdocs\ci4App\system\CodeIgniter.php(832): App\Controllers\UserController->login()
#10 C:\xampp\htdocs\ci4App\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#11 C:\xampp\htdocs\ci4App\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4App\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ci4App\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
